Cubical/Categories/NaturalTransformation.agda

```diff
@@ -8,72 +8,79 @@ open import Cubical.Categories.Functor
-  seq-trans α β .N-ob x = 𝒟 .seq (α .N-ob x) (β .N-ob x)
-  seq-trans {F} {G} {H} α β .N-hom f =
-    𝒟 .seq (F .F-hom f) (𝒟 .seq (α .N-ob _) (β .N-ob _))
-      ≡⟨ sym (𝒟 .seq-α _ _ _) ⟩
-    𝒟 .seq (𝒟 .seq (F .F-hom f) (α .N-ob _)) (β .N-ob _)
-      ≡[ i ]⟨ 𝒟 .seq (α .N-hom f i) (β .N-ob _) ⟩
-    𝒟 .seq (𝒟 .seq (α .N-ob _) (G .F-hom f)) (β .N-ob _)
-      ≡⟨ 𝒟 .seq-α _ _ _ ⟩
-    𝒟 .seq (α .N-ob _) (𝒟 .seq (G .F-hom f) (β .N-ob _))
-      ≡[ i ]⟨ 𝒟 .seq (α .N-ob _) (β .N-hom f i) ⟩
-    𝒟 .seq (α .N-ob _) (𝒟 .seq (β .N-ob _) (H .F-hom f))
-      ≡⟨ sym (𝒟 .seq-α _ _ _) ⟩
-    𝒟 .seq (𝒟 .seq (α .N-ob _) (β .N-ob _)) (H .F-hom f)
+  seqTrans : {F G H : Functor C D} (α : NatTrans F G) (β : NatTrans G H) → NatTrans F H
+  seqTrans α β .N-ob x = (α .N-ob x) ⋆ᴰ (β .N-ob x)
+  seqTrans {F} {G} {H} α β .N-hom f =
+    (F .F-hom f) ⋆ᴰ ((α .N-ob _) ⋆ᴰ (β .N-ob _))
+      ≡⟨ sym (D .seq-α _ _ _) ⟩
+    ((F .F-hom f) ⋆ᴰ (α .N-ob _)) ⋆ᴰ (β .N-ob _)
+      ≡[ i ]⟨ (α .N-hom f i) ⋆ᴰ (β .N-ob _) ⟩
+    ((α .N-ob _) ⋆ᴰ (G .F-hom f)) ⋆ᴰ (β .N-ob _)
+      ≡⟨ D .seq-α _ _ _ ⟩
+    (α .N-ob _) ⋆ᴰ ((G .F-hom f) ⋆ᴰ (β .N-ob _))
+      ≡[ i ]⟨ (α .N-ob _) ⋆ᴰ (β .N-hom f i) ⟩
+    (α .N-ob _) ⋆ᴰ ((β .N-ob _) ⋆ᴰ (H .F-hom f))
+      ≡⟨ sym (D .seq-α _ _ _) ⟩
+    ((α .N-ob _) ⋆ᴰ (β .N-ob _)) ⋆ᴰ (H .F-hom f)
       ∎
 
-  module _  ⦃ 𝒟-category : isCategory 𝒟 ⦄ {F G : Functor 𝒞 𝒟} {α β : NatTrans F G} where
+  module _  ⦃ D-category : isCategory D ⦄ {F G : Functor C D} {α β : NatTrans F G} where
     open Precategory
     open Functor
     open NatTrans
 
-    make-nat-trans-path : α .N-ob ≡ β .N-ob → α ≡ β
-    make-nat-trans-path p i .N-ob = p i
-    make-nat-trans-path p i .N-hom f = rem i
+    makeNatTransPath : α .N-ob ≡ β .N-ob → α ≡ β
+    makeNatTransPath p i .N-ob = p i
+    makeNatTransPath p i .N-hom f = rem i
       where
-        rem : PathP (λ i → 𝒟 .seq (F .F-hom f) (p i _) ≡ 𝒟 .seq (p i _) (G .F-hom f)) (α .N-hom f) (β .N-hom f)
-        rem = toPathP (𝒟-category .homIsSet _ _ _ _)
+        rem : PathP (λ i → (F .F-hom f) ⋆ᴰ (p i _) ≡ (p i _) ⋆ᴰ (G .F-hom f)) (α .N-hom f) (β .N-hom f)
+        rem = toPathP (D-category .homIsSet _ _ _ _)
 
 
-module _ (𝒞 : Precategory ℓ𝒞 ℓ𝒞') (𝒟 : Precategory ℓ𝒟 ℓ𝒟') ⦃ _ : isCategory 𝒟 ⦄ where
+module _ (C : Precategory ℓC ℓC') (D : Precategory ℓD ℓD') ⦃ _ : isCategory D ⦄ where
   open Precategory
   open NatTrans
   open Functor
 
-  FUNCTOR : Precategory (ℓ-max (ℓ-max ℓ𝒞 ℓ𝒞') (ℓ-max ℓ𝒟 ℓ𝒟')) (ℓ-max (ℓ-max ℓ𝒞 ℓ𝒞') (ℓ-max ℓ𝒟 ℓ𝒟'))
-  FUNCTOR .ob = Functor 𝒞 𝒟
-  FUNCTOR .hom = NatTrans
-  FUNCTOR .idn = id-trans
-  FUNCTOR .seq = seq-trans
-  FUNCTOR .seq-λ α = make-nat-trans-path λ i x → 𝒟 .seq-λ (α .N-ob x) i
-  FUNCTOR .seq-ρ α = make-nat-trans-path λ i x → 𝒟 .seq-ρ (α .N-ob x) i
-  FUNCTOR .seq-α α β γ = make-nat-trans-path λ i x → 𝒟 .seq-α (α .N-ob x) (β .N-ob x) (γ .N-ob x) i
+  FUNCTOR : Precategory (ℓ-max (ℓ-max ℓC ℓC') (ℓ-max ℓD ℓD')) (ℓ-max (ℓ-max ℓC ℓC') (ℓ-max ℓD ℓD'))
```

Comment:
Maybe we should call this `FunctorPrecat`?

Reply:
Same comment as for SET and TYPE, isn't all caps standard?

Reply:
No? Both Wikipedia and the nlab uses "Set"

Also, it's better if the name reflects that this is the `Precat` version. There should also be a category and univalent category version of this.

Reply:
Hmm yeah you're right; Riehl's book uses **SET** in place of **Set** for when things aren't "small." But I looked it up too and don't see the notation elsewhere.

When you say there will be category and univalent versions, does that mean the versions will actually be different categories? For example, isn't **SET** as defined in `Sets.agda` already a Category, be we just haven't proven that it is an instance of `isCategory` yet.

Reply:
Ah, I was confused and thought that precategory, category and univalent category all had different definitions. But the way things are currently done `isCategory` and `isUnivalent` are just predicates. `

Reply:
Right right. Are you good with the names as is then? Or would you prefer something like FunctorCat/SetCat?